textattack/commands/attack/attack_args_helpers.py

```diff
@@ -414,28 +414,54 @@ def parse_dataset_from_args(args):
 def parse_logger_from_args(args):
     # Create logger
     attack_log_manager = textattack.loggers.AttackLogManager()
-    out_time = int(time.time() * 1000)  # Output file
-    # Set default output directory to `textattack/outputs`.
-    if not args.out_dir:
-        current_dir = os.path.dirname(os.path.realpath(__file__))
-        outputs_dir = os.path.join(
-            current_dir, os.pardir, os.pardir, os.pardir, "outputs", "attacks"
-        )
-        if not os.path.exists(outputs_dir):
-            os.makedirs(outputs_dir)
-        args.out_dir = os.path.normpath(outputs_dir)
+
+    # Get current time for file naming
+    year, month, day, hour, minute = map(str, time.strftime("%Y %m %d %H %M").split())
```

Comment:
these two lines can be condensed into: `timestamp = time.strftime("%Y-%m-%d-%H-%M")`